docs/core/diagnostics/distributed-tracing-builtin-activities.md

```diff
@@ -0,0 +1,151 @@
+---
+title: Built-in Activities in .NET
+description: Overview of activities emitted by the .NET libraries
+ms.date: 12/05/2024
+---
+
+# Built-in Activities in .NET
+
+This is a reference for distributed tracing Activities (<xref:System.Diagnostics.Activity>) emitted natively by .NET-s built-in <xref:System.Diagnostics.ActivitySource> instances.
+
+## System.Net Activities
+
+### HTTP client request
+
+<xref:System.Net.Http.SocketsHttpHandler> and <xref:System.Net.Http.HttpClientHandler> report the the HTTP client request activity following the recommendations defined in OpenTelemetry [HTTP Client Semantic Conventions](https://opentelemetry.io/docs/specs/semconv/http/http-spans/#http-client). It describes the HTTP request sent by <xref:System.Net.Http.HttpClient>'s send overloads during the time interval the underlying handler completes the request. Completing the request includes the time up to reading response headers from the network stream. It doesn't include the time spent reading the response body.
```

Comment:
Starting from this point I'm intentionally using `activity` as a lower-case common noun. The word is used so often (many times in plural) that referencing the `Activity` class all the time would be IMO confusing.

Reply:
Would it make sense to say `activity object`?

Reply:
> Would it make sense to say `activity object`? 

Feels too verbose to me. Also, I prefer to talk about *activity* as a generic term, a synonym of OTel span. The `Activity` .NET object is just one representation, in the backend it will be a database record.